File: src/main/Java/boj_step3/boj2439.java
import java.io.*;

public class boj2439 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        int i = 1;
        while (i <= n) {
            int j = 1;
            int k = n - i;
            while (k-- > 0) writer.append(" ");
            while (j++ <= i) {
                writer.append("*");
            }
            writer.newLine();
            i++;
        }
        writer.close();
    }
}

File: src/main/kotlin/others/boj2439.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    for (i in 1..n) {
        for (j in n - 1 downTo  i) w.write(" ")
        for (j in 1..i) w.write("*")
        w.write("\n")
    }
    w.close()
}